wallet/src/Pos/Wallet/Aeson/ClientTypes.hs

```diff
@@ -53,15 +54,26 @@ instance ToJSON (CBackupPhrase mw) where
     toJSON =
         genericToJSON defaultOptions
 
+backwardsCompitability :: [Text] -> [Text]
+backwardsCompitability mnemonicWords = mnemonicWords >>= splitWords
+    where
+        splitWords :: Text -> [Text]
+        splitWords = split (== ' ')
```

Comment:
why not https://hackage.haskell.org/package/text-1.2.3.0/docs/Data-Text.html#v:words ?

Reply:
👍 thanks!